### ISSUE TITLE
Async not working

### ISSUE THREAD

**Issue by shdblowers** [meadsteve/white-bread#71]
Hi,

Am trying to run my suites async, as so:
```
  suite name:          "User",
        context:       Example.UserContext,
        feature_paths: ["features/user/"],
        async:         true
```

But I get the following output:
```
loading config from features/config.exs
** (KeyError) key :async not found
    (elixir) lib/map.ex:515: Map.update!/3
    (elixir) lib/enum.ex:1623: Enum."-reduce/3-lists^foldl/2-0-"/3
    features/config.exs:4: (module)
    (stdlib) erl_eval.erl:670: :erl_eval.do_apply/6

```

Am running white_bread version "2.8.1"

**Comment by shdblowers** [comment]
Just had a look through some closed issues, can see in there `:run_async` is used as the key.

I was following the documentation on the `README.md`, which is incorrect.

**Comment by meadsteve** [comment]
@shdblowers thanks for highlighting this. I'll update the README.

**Comment by meadsteve** [comment]
fixed in 7c2aa914bfb3c407a47878926f99fccfaabf9c87
